packages/lodestar/src/network/reqresp/types.ts

```diff
@@ -141,20 +141,33 @@ export function getResponseSzzTypeByMethod(protocol: Protocol, forkName: ForkNam
   }
 }
 
-export type ResponseBodyByMethod = {
+type CommonResponseBodyByMethod = {
   [Method.Status]: phase0.Status;
   [Method.Goodbye]: phase0.Goodbye;
   [Method.Ping]: phase0.Ping;
   [Method.Metadata]: phase0.Metadata;
+};
+
+// Used internally by lodestar to response to beacon_blocks_by_range and beacon_blocks_by_root
+// without having to deserialize and serialize from/to bytes
+export type LodestarResponseBodyByMethod = CommonResponseBodyByMethod & {
```

Comment:
Instead of Lodestar you can denote this with `Incoming` vs `Outgoing` responses.
- On an `Incoming` response we deserialize
- On an `Outgoing` response we do not deserialize